README.md

```diff
@@ -72,6 +73,13 @@
   * [drf-api-tracking](https://github.com/lingster/drf-api-tracking): Provides a Django model and DRF view mixin that work together to log Django Rest Framework requests to the database.
   * [django-requestlogs](https://github.com/Raekkeri/django-requestlogs): Package providing middleware and other helpers for audit logging.
 
+  ### Filtering
+  
+  * [django-rest-framework-filters](https://github.com/philipn/django-rest-framework-filters): Better filtering.
```

Comment:
https://github.com/philipn/django-rest-framework-filters seems abandoned. Not sure if adding it helps.